arbitrum-docs/run-arbitrum-node/nitro/03-nitro-database-snapshots.md

```diff
@@ -0,0 +1,148 @@
+---
+title: 'Nitro database snapshots'
+description: This page explains how to supply a snapshot to Nitro and how to create a new snapshot
+author: gligneul
+sme: gligneul
+target_audience: 'Node runners interested in starting Nitro with a database snapshot'
+content_type: how-to
+sidebar_position: 11
+---
+
+Nitro stores the chain state and data in a database in the local filesystem. When starting Nitro for the first time, it will initialize an empty database by default and start processing transactions from genesis. It takes a long time for the node to sync from genesis, so starting from a database snapshot is advisable instead. Moreover, for the Arb1 chain, you must start from a snapshot because Nitro cannot process transactions from the Classic Arbitrum node.
```

Comment:
```suggestion
Nitro stores the chain state and data in a database in the local filesystem. When starting Nitro for the first time, it will initialize an empty database by default and start processing transactions from Genesis. It takes a long time for the node to sync from Genesis, so starting from a database snapshot is advisable instead. Moreover, for the Arb1 chain, you must start from a snapshot because Nitro cannot process transactions from the Classic Arbitrum node.
```